TodoApp/TodoApp.Api/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web.Http;
+using System.Web.Http.Description;
+using Microsoft.Web.Http;
+using TodoApp.Api.Models;
+
+namespace TodoApp.Api.Controllers
+{
+    [ApiVersion("1.0")]
+    [RoutePrefix("api/v{version:apiVersion}/items")]
+    public class ItemsController : ApiController
+    {
+        internal static readonly IList<Item> Items = new List<Item>
+        {
+            new Item
+            {
+                Id = "1",
+                Text = "Learn react"
+            },
+            new Item
+            {
+                Id = "2",
+                Text = "Learn redux"
+            },
+            new Item
+            {
+                Id = "3",
+                Text = "Write Web API"
+            }
+        };
+
+        // GET: api/v1/items
+        [Route]
+        public async Task<IEnumerable<Item>> GetAllItemsAsync()
+            => await Task.FromResult(Items);
+
+        // GET: api/v1/items/5
+        [Route("{id}")]
+        public async Task<IHttpActionResult> GetItemByIdAsync(string id)
+            => await Task.FromResult(Ok(Items.FirstOrDefault(i => i.Id == id)));
+        
+
+        // POST: api/v1/items
+        [Route("", Name = "PostNewItem")]
+        public async Task<IHttpActionResult> PostItemAsync([FromBody]Item item)
+        {
+            Items.Add(item);
+            return await Task.FromResult(CreatedAtRoute("PostNewItem", new { id = item.Id }, item));
+        }
+
+        // PUT: api/v1/items/5
+        [Route("{id}")]
+        public async Task<IHttpActionResult> PutItemAsync(string id, [FromBody]Item value)
+        {
+            var item = Items.FirstOrDefault(i => i.Id == id);
```

Comment:
In this very first task, we implement just a dummy controller to implement the prototype of the rest API. Because of that, you should implement just basic methods for the POST, PUT, GET, DELETE - these methods should have no business logic - returning one specific item for POST, the second one for PUT etc. is perfectly fine.